src/main/resources/application.yml

```diff
@@ -0,0 +1,5 @@
+# Show details of health endpoint
+management.endpoint.health.show-details: always
+
+host:
+  name: https://nofluffjobs.com/pl/praca-it/krakow/backend
```

Comment:
Hostname in this case is 'nofluffjobs.com'. Split this configuration to baseUrl (https://nofluffjobs.com) and endpoint(/pl/praca-it/krakow/backend).